db/migrate/20251104150435_add_course_code_to_chaser.rb

```diff
@@ -0,0 +1,5 @@
+class AddCourseCodeToChaser < ActiveRecord::Migration[8.0]
+  def change
+    add_column :chasers_sent, :course_code, :string, null: true
```

Comment:
Would course id be better since course_code is not uniq? (it's a combination of recruitment_cycle_year, provider_id, and course_code that is unique?)